src/components/ListItem/ListItem.test.js

```diff
@@ -89,7 +89,19 @@ describe('ListItem', () => {
       expect(toggleNotExists).toBeTruthy();
     });
 
-    it('Should be visible if hasToggle is  provided', () => {
+    it('Should not visible if hasCheckbox is provided', () => {
```

Comment:
(nit, sp) Should be: `Should not be visible...`

Similar comment for line: 98, below.